packages/app/src/cli/utilities/developer-platform-client/app-management-client.test.ts

```diff
@@ -185,7 +185,7 @@ describe('templateSpecifications', () => {
     // When
     const client = new AppManagementClient()
     client.businessPlatformToken = () => Promise.resolve('business-platform-token')
-    const got = await client.templateSpecifications(orgApp)
+    const {templates: got} = await client.templateSpecifications(orgApp)
```

Comment:
I think we should have some test in this file to validate that we've extracted the groupOrder correctly